pageserver/src/tenant/tasks.rs

```diff
@@ -375,6 +375,9 @@ async fn gc_loop(tenant: Arc<Tenant>, cancel: CancellationToken) {
                     .gc_iteration(None, gc_horizon, tenant.get_pitr_interval(), &cancel, &ctx)
                     .await;
                 if let Err(e) = res {
+                    if cancel.is_cancelled() {
+                        break;
+                    }
```

Comment:
This seems unrelated to the PR?

Reply:
oops.  Harmless but unintended for this PR.  49666770f